apps/ledger-live-desktop/tests/specs/speculos/accountSpeculos.spec.ts

```diff
@@ -0,0 +1,91 @@
+import test from "../../fixtures/common";
+import { expect } from "@playwright/test";
+import { PortfolioPage } from "../../models/PortfolioPage";
+import { AddAccountModal } from "../../models/AddAccountModal";
+import { Layout } from "../../models/Layout";
+import { AccountPage } from "../../models/AccountPage";
+import { AccountsPage } from "../../models/AccountsPage";
+import { Device, specs, startSpeculos, stopSpeculos } from "../../utils/speculos";
+import { Currency } from "../../enum/Currency";
+
+test.use({ userdata: "skip-onboarding" });
+const currencies: Currency[] = [
+  Currency.BTC,
+  Currency.tBTC,
+  Currency.ETH,
+  Currency.tETH,
+  Currency.SOL,
+  Currency.DOT,
+  Currency.TRX,
+  Currency.XRP,
+];
+
+let device: Device | undefined;
```

Comment:
(if i recall we worked together on a way to not need this)
to recap: since device is shared between parallel runs, you can't share such variable and expect that stopSpeculos(device) will work. maybe not a visible problem (since there is apparently a clean up of zombie speculos docket instance at some point anyway), but essentially the stopSpeculos is not correctly implemented today / is probably just working on ONE of the parallel run.

Reply:
I cherry-picked the code we worked on together last week; it has been removed now. 😁